### PR TITLE
feat(backend)!: Rename id to uuid and alias to eid everywhere

### PR DESCRIPTION
**Pull request by samiralavi**
This big PR is all about renaming id to uuid and alias to eid. I guess my eyes's number increased by 1 as I tried to find out every usage and rename them. VSCode Python auto-rename function helped a lot though.

### CODE REVIEW

File: aqueductcore/backend/models/experiment.py [hunk 1]
```diff
@@ -29,23 +29,22 @@ class TagRead(TagBase):
...
 
-    alias: str
+    eid: str
     tags: List[TagCreate]
 
 
 class ExperimentRead(ExperimentBase):
     """Model fields to read an experiment"""
 
-    id: UUID
```
Comment (str-anger) [inline aqueductcore/backend/models/experiment.py:47]: Is this intentionally you drop UUID?

File: aqueductcore/backend/models/experiment.py [hunk 2]
```diff
@@ -59,5 +58,5 @@ class ExperimentUpdate(ExperimentBase):
 class UserBase(AQDModel):
     """Base model for User"""
 
-    user_id: UUID
+    uuid: UUID
```
Comment (str-anger) [inline aqueductcore/backend/models/experiment.py:61]: in `UserInfo` you use `user_uuid` name.

File: aqueductcore/backend/models/plugin.py [hunk 3]
```diff
@@ -91,10 +91,10 @@ def validate_value(self, value: str) -> str:
             try:
                 prefix, postfix = value.split("-")
                 if not prefix.isdecimal() or not postfix.isalnum():
-                    raise AQDValidationError("Experiment alias has wrong format.")
+                    raise AQDValidationError("Experiment EID has wrong format.")
```
Comment (str-anger) [inline aqueductcore/backend/models/plugin.py:94]: EID is already Experiment ID :)

File: aqueductcore/backend/routers/graphql/mutations_schema.py [hunk 4]
```diff
@@ -128,7 +131,7 @@ async def execute_plugin(
         exp_parameter = function_object.get_default_experiment_parameter()
         if exp_parameter is None:
             raise AQDValidationError(f"Function {plugin}/{function} has no experiment parameters")
-        exp_id = dict_params[exp_parameter.name]
+        eid = dict_params[exp_parameter.name]
```
Comment (str-anger) [inline aqueductcore/backend/routers/graphql/mutations_schema.py:134]: Hmmmm, but okay =)

File: aqueductcore/backend/services/experiment.py [hunk 5]
```diff
@@ -245,7 +245,7 @@ async def get_experiment_files(
     return file_names
 
 
-EXPERIMENT_ALIAS_PATTERN = r"^(19[0-9]{2}|2[0-9]{3})(0[1-9]|1[012])([123]0|[012][1-9]|31)-(\d+)$"
+EXPERIMENT_EID_PATTERN = r"^(19[0-9]{2}|2[0-9]{3})(0[1-9]|1[012])([123]0|[012][1-9]|31)-(\d+)$"
```
Comment (str-anger) [inline aqueductcore/backend/services/experiment.py:248]: EID is already Experiment ID

**Review by str-anger — APPROVED** [review]
I'm probably ok. Some minor comments of naming

File: aqueductcore/backend/models/experiment.py (hunk 1 above)
Comment (samiralavi) [reply aqueductcore/backend/models/experiment.py:47]: Yes, it was a kind of bug as the `id` was already defined in the base class.

File: aqueductcore/backend/models/experiment.py (hunk 2 above)
Comment (samiralavi) [reply aqueductcore/backend/models/experiment.py:61]: yeah now I look at it, it is better to change that to `uuid` instead.

File: aqueductcore/backend/models/plugin.py (hunk 3 above)
Comment (samiralavi) [reply aqueductcore/backend/models/plugin.py:94]: I know it is repetitive but I wanted to make sure everywhere we are talking about the ID, the term EID and UUID is clearly used to denote that.

File: aqueductcore/backend/services/experiment.py (hunk 5 above)
Comment (samiralavi) [reply aqueductcore/backend/services/experiment.py:248]: Yeah, I didn't want to use `ID` on its own without any prefix anywhere to make sure the reference is clear

**Review by jatinriverlane — APPROVED** [review]
Good job Amir

**Pull request merged**